testsuite/tests/singlecluster/defaults/merge/same_target/test_ab_strategy.py

```diff
@@ -0,0 +1,27 @@
+"""Test defaults policy aimed at the same resoure uses oldested policy."""
+
+import pytest
+
+from .conftest import MERGE_LIMIT, MERGE_LIMIT2
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def commit(request, route, rate_limit, default_merge_rate_limit):  # pylint: disable=unused-argument
+    """Commits RateLimitPolicy after the HTTPRoute is created"""
+    for policy in [rate_limit, default_merge_rate_limit]:  # Forcing order of creation.
+        request.addfinalizer(policy.delete)
+        policy.commit()
+        policy.wait_for_accepted()
+
+
+def test_multiple_policies_merge_default_ab(client):
+    """Test RateLimitPolicy with merge defaults being ignored due to age"""
+    responses = client.get_many("/get", MERGE_LIMIT.limit)
```

Comment:
To be honest I think that the comments need to be swapped between test_ab_strategy and test_ba_strategy. If I got it right "a" means standard policy and "b" means policy with merge defaults.

So in "ab" the "a" policy is created first, and once "b" is created the "a" gets overridden - you can see it in test that "MERGE_LIMIT" is used for both `get_many` calls. However, the comment in "ab" suggests that "b" is ignored which is not true - it is true in the "ba" test where the policies are created in different order.

Reply:
I decrypted the policy names.
`dmp` means policy with defauls section and merge strategy (defaults merge policy)
`sp` means standard policy === policy without both defaults and overrides sections
`omp` means policy with overrides section and merge strategy (overrides merge policy)